rust/rope/src/tree.rs

```diff
@@ -45,37 +45,44 @@ pub trait NodeInfo: Clone {
 
     /// The identity of the monoid. Need not be implemented because it
     /// can be computed from the leaf default.
+    /// 
+    /// TODO: Why have it if it never needs to be overridden?
```

Comment:
It might be worth keeping this for pedagogical reasons. The identity element is one of the fundamental properties of a monoid. It _needn't_ be overridden, but in many monoids it has a simple value (0 for anything that counts). I think it will also be important for property testing.